### PR TITLE
Fixing error/verbose messages & SecondaryDNS fix

### PR DESCRIPTION
**Pull request by therealsanmah**
Fixing the error and verbose messages in the resource files in accordance with the scrubbed messages provided by Harsha.

### CODE REVIEW

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 1]
```diff
@@ -193,73 +193,73 @@
...
-    <value>More backups are available in the subsequent pages for your query. To access the next page use \"-Skip {0}\"  in your commandlet</value>
+    <value>More backups are available in subsequent pages for your query. To access the next page, use \"-Skip {0}\" in your cmdlet.</value>
   </data>
   <data name="BackupNoMorePagesMessage" xml:space="preserve">
-    <value>No more backup sets are present for your query!</value>
+    <value>No more backup sets are available for your query.</value>
   </data>
   <data name="FirstParameterInvalidMessage" xml:space="preserve">
-    <value>Parameter First cannot be &lt;0</value>
+    <value>The First parameter cannot be &lt;0.</value>
```
Comment (Sumanth-SK) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Consider modifying to "The 'First' parameter cannot be ..." or "The parameter 'First' cannot be ..."

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 2]
```diff
@@ -415,58 +415,58 @@
     <value>Could not persist secret</value>
   </data>
   <data name="DeviceNotConfiguredMessage" xml:space="preserve">
-    <value>The device name you have specified is not yet configured fully. Please complete the configuration and retry.</value>
+    <value>The device name is not fully configured. Please complete the configuration and retry.</value>
```
Comment (Sumanth-SK) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Consider modifying this, as it gives the impression that the 'device name' is not configured correctly, whereas the message is about the 'device with specified device name' is not configured correctly.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 3]
```diff
@@ -415,58 +415,58 @@
...
   </data>
   <data name="DeviceNotRegisteredMessage" xml:space="preserve">
-    <value>No StorSimple device is currently registered with this resource. Please register at least one device to the resource and rerun this command.</value>
+    <value>No StorSimple device is currently registered with this resource. Please register at least one device with the resource and then rerun this command.</value>
   </data>
   <data name="ResourceContextNotSetMessage" xml:space="preserve">
-    <value>Resource Context is not set for your subscription. Please use Select-AzureStorSimpleResource -ResourceName &lt;&lt;name&gt;&gt; to set the resource context.</value>
+    <value>The resource context is not set for your subscription. Please use Select-AzureStorSimpleResource -ResourceName &lt;&lt;name&gt;&gt; to set the resource context.</value>
```
Comment (Sumanth-SK) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Resource Context is not something the user needs to understand.
So, consider modifying to - "The Azure StorSimple Resource is not set for your subscription..."

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx [hunk 4]
```diff
@@ -415,58 +415,58 @@
...
-    <value>More jobs are available in the subsequent pages for your query. To access the next page use -Skip {0}  in your commandlet</value>
+    <value>More jobs are available in subsequent pages for your query. To access the next page, use -Skip {0} in your cmdlet.</value>
   </data>
   <data name="DeviceJobsNoMorePagesMessage" xml:space="preserve">
-    <value>No more jobs are present for your query!</value>
+    <value>No more jobs are available for your query.</value>
   </data>
   <data name="DeviceJobsReturnedCount" xml:space="preserve">
-    <value>{0} Job{1} found!</value>
+    <value>Found {0} job{1}.</value>
```
Comment (Sumanth-SK) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: space between job and {1}?

Comment (therealsanmah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Not required - the {1} is a modifier for plural form when the value of {0} is greater than 1. Also, tbh, the review is not really for the messages. All these messages have been provided/reviewed by our content team.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx (hunk 2 above)
Comment (therealsanmah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: I do agree to this one though. Seems a valid point.

File: src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx (hunk 1 above)
Comment (therealsanmah) [inline src/ServiceManagement/StorSimple/Commands.StorSimple/Properties/Resources.resx]: Hence the capitalization on the First.

**Pull request merged**